### DISCUSSION TITLE
Qiitadon および Qiitan（キーたん）の名称の取り扱いについて

### DISCUSSION THREAD

**Discussion by KEINOS** [increments/qiita-discussions#132] [ヘルプ] [answered]
> [Qiitadonのサービス提供を終了します](https://blog.qiita.com/terminate-provision-of-qiitadon/) @ Qiita Blog

おそらく大半の Qiitadon ユーザは理由を理解・納得できていると思います。

むしろ、企業インスタンスが続々と消えていくなか、永らく提供いただき、ありがとうございました。Qiita と Qiitadon で育った身としては感謝しかありません。

さて、**ファンによる Qiitadon および Qiitan（キーたん）の名称の利用についての確認**があります。

## Qiitadon

サービス終了（インスタンスの閉鎖）に向けて別インスタンスに各々散らばりはじめているのですが、各種アーカイブ先および同窓の場として急遽以下のドメインとリポジトリを取得いたしました。

- https://qiitadon.fans/
- https://github.com/qiitadon

「Qiita 非公認の Qiitadon ファンによるもの」を明記することを大前提として、この場合の Qiitadon の名称利用に Qiita として困ることはありますでしょうか。

## Qiitan（キーたん）

サービス終了（インスタンスの閉鎖）後、「元 Qiitadon メンバー」および「キーたんファン」の共通言語として、「キーたん語」というスクリプト言語をリリース予定（現在クローズド β）です。（たいしたスクリプト言語ではないのですが）

その場合も「Qiita 非公認の、キーたんファンによるスクリプト言語」という前提で、Qiitan の名称利用に Qiita として困ることはありますでしょうか。

もしろん、どちらの利用も「暴力的な表現、反社会的な勢力を想起させる、又は、性的な表現に該当し公序良俗に反する恐れがあるもの」および「名称を汚すような目的」ではないことを誓います。

**Comment by kskwtnk** [comment, marked as answer]
@KEINOS 
Discussionを立てていただいて & Qiitadonをご利用いただいてありがとうございます！

こちらの件ですが、社内で確認いたしますが少し時間がかかってしまうかもしれません 🙇 
正式な回答は追ってこちらのDiscussionに返信いたします！申し訳ありませんが少々お待ちください 🙇

**Reply by KEINOS** [reply]
お返事ありがとうございます！お手数ですが、よろしくお願いいたします。🙇

**Reply by kskwtnk** [reply]
@KEINOS 
大変遅くなって申し訳ありません！

結論、OKです！
なのですが、以下をご理解のうえで運営頂ければと思います 🙏 

- あくまで非公式ということでの運営をお願いします
- 弊社の営業に望ましくない影響が生じると判断した場合には、当該内容へのご対応依頼等、必要に応じたご対応をさせて頂く場合もあります

特に2点目は堅苦しいというか仰々しいのですが、ご了承ください 🙏

**Comment by KEINOS** [comment]
@xrxoxcxox 

> 結論、OKです！
> なのですが、以下をご理解のうえで運営頂ければと思います 🙏
>
> - **あくまで非公式**ということでの運営をお願いします
> - **弊社の営業に望ましくない影響が生じると判断した場合**には、当該内容へのご対応依頼等、必要に応じたご対応をさせて頂く場合もあります

ありがとうございます！

キーたんと Qiitadon のファンクラブ（同窓会？）であるため、非公式はもちろんのこと、Qiita の営業妨害をするつもりも毛頭ありません。支障がある場合は、遠慮なく注意いただければと存じます。

年度末が近いお忙しい中、確認いただきありがとうございました。改めまして、永い間 Qiitadon を提供くださりありがとうございました！
